package.json

```diff
@@ -62,6 +63,7 @@
     "babel-plugin-prismjs": "2.1.0",
     "babel-plugin-styled-components": "2.0.7",
     "css-loader": "6.7.1",
+    "cypress": "^10.4.0",
```

Comment:
We'll have a renovabot MR following to fix the version. BTW not sure if we should 🤔 but that's a different topic ˆˆ

You could fix it in this PR, as you wish